### PR TITLE
Support django 1.4 and django 1.8

### PR DESCRIPTION
**Pull request by jlward**

### CODE REVIEW

File: announcements/admin.py [hunk 1]
```diff
@@ -25,4 +25,7 @@ def save_model(self, request, obj, form, change):
         obj.save()
 
 
-admin.site.register(Announcement, AnnouncementAdmin)
+try:
+    admin.site.register(Announcement, AnnouncementAdmin)
```
Comment (winhamwr) [inline announcements/admin.py]: Why is this necessary? Is it normal for apps to do this?

File: announcements/tests.py [hunk 2]
```diff
@@ -1,35 +1,74 @@
...
->>> a2 = Announcement.objects.create(title="Down for Maintenance Again", creator=superuser)
->>> a3 = Announcement.objects.create(title="Down for Maintenance Again And Again", creator=superuser, site_wide=True)
->>> a4 = Announcement.objects.create(title="Members Need to Fill Out New Profile Info", creator=superuser, members_only=True)
->>> a5 = Announcement.objects.create(title="Expected Down Time", creator=superuser, members_only=True, site_wide=True)
 
-# get the announcements that are publically viewable. this is the same as
-# calling as using site_wide=False, for_members=False
->>> Announcement.objects.current()
-[<Announcement: Down for Maintenance Again And Again>, <Announcement: Down for Maintenance Again>, <Announcement: Down for Maintenance>]
+class AnnouncementTestCase(TestCase):
```
Comment (winhamwr) [inline announcements/tests.py]: Death to doctests :)

File: announcements/urls.py [hunk 3]
```diff
@@ -1,16 +1,23 @@
-from django.conf.urls.defaults import *
-from django.views.generic import list_detail
+try:
+    from django.views.generic.list_detail import object_detail as DetailView
+except ImportError:  # Django > 1.4
```
Comment (winhamwr) [inline announcements/urls.py]: :+1: thanks for the code comments

File: announcements/views.py [hunk 4]
```diff
@@ -17,7 +24,7 @@ def announcement_list(request):
     announcements.
     """
     queryset = current_announcements_for_request(request)
-    return list_detail.object_list(request, **{
+    return ListView(request, **{
```
Comment (winhamwr) [inline announcements/views.py]: What's up with the weird `**{` syntax? Can't you just do:
```python
return ListView(request, queryset=queryset, allow_empty=True)
```

**Comment by winhamwr** [comment]
Could we add a `policystat_changelog.md` file for this PR and add this:

```markdown
# Changes in PolicyStat's fork

* Added tox tests and travis integration
* Added support for Django 1.8 while keeping Django 1.4 support
* Removed the feeds feature. PolicyStat was not using it.
```

When future-us look at our fork and try to figure out what's up, that will really help (versus working through the pull requests).

File: announcements/admin.py (hunk 1 above)
Comment (jlward) [inline announcements/admin.py]: I've seen it in other places.

File: policystat_changelog.md [hunk 5]
```diff
@@ -0,0 +1,5 @@
+# Changes in PolicyStat's fork
+
+* Added tox tests and travis integration
```
Comment (winhamwr) [inline policystat_changelog.md]: Also:
* Added python3.3 support

**Comment by winhamwr** [comment]
:+1:

File: .travis.yml [hunk 6]
```diff
@@ -2,7 +2,9 @@ language: python
 python:
   - "2.7"
 env:
-  - TOX_ENV=py26
```
Comment (bocribbz) [inline .travis.yml]: ah lol nvm my other comment it got fixed here

File: announcements/urls.py (hunk 3 above)
Comment (bocribbz) [inline announcements/urls.py]: I must say that i do prefer the other way around,
```
try:
    new version
except:
    old version
```
